trackers/huggingface/app.py

```diff
@@ -21,10 +21,44 @@ def add_faq() -> None:
     """
     faq = Collapsable()
```

Comment:
Add a section for the experimental setup. What specific devices were used, in what cloud, with what software version numbers? What is the benchmarking methodology (can link to the benchit user guide). Mention that benchit was used to collect the benchmarks.

Reply:
Added methodology sections. We really have to be careful here to always update that section.

Reply:
Yeahhh keeping all of that up to date will be effort-intensive if we do it manually, and super bad if we ever forget. 

Filed https://github.com/groq/mlagility/issues/189 to address this later